package.json

```diff
@@ -4,7 +4,7 @@
 	"scripts": {
 		"build": "pnpm -r build",
 		"start": "pnpm --filter main start",
-		"start:dev": "pnpm --filter main start:dev",
+		"start:dev": "pnpm -r build && pnpm --filter main start:dev",
```

Comment:
What do we need to build if dev is still `tsx` ?

Reply:
changes made outside of the main package are not reflected with the current ts config.

Reply:
Please check now, I changed it to force using `tsx` for all packages.

Reply:
Also, you can continue with the remaining assigned tasks in the repo afterward.

Reply:
looks good, thanks.